pkg/app/impl/authz.go

```diff
@@ -55,10 +58,18 @@ func NewAuthorizerServer(
 ) *AuthorizerServer {
 	newLogger := logger.With().Str("component", "api.grpc").Logger()
 
+	const certs = `https://citadel.demo.aserto.com/dex/keys`
+
+	// TODO: get context
```

Comment:
I assume the lifetime of the cache is equal to the process its lifetime, and there are no cancellation or deadline requirements,  making context.Background() an acceptable solution, am I missing something?